package.json

```diff
@@ -128,7 +128,7 @@
     "ember-resolver": "^8.0.3",
     "ember-route-action-helper": "^2.0.8",
     "ember-router-scroll": "^3.3.7",
-    "ember-simple-auth": "^4.2.2",
+    "ember-simple-auth": "^5.0.0",
```

Comment:
we can remove the auth version 4.0.0 since the code would work fine then also.

Reply:
@jpsxlr8 please create PR to remove package if not needed